### PR TITLE
Test structure

### PR DESCRIPTION
**Pull request by MStraatman**
Implemented a folder structure for the tests. Also increased the coverage yielded by NodeTest and made GraphSceneTest(yielding 88% line coverage)

### CODE REVIEW

File: src/main/java/screens/Factory.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package screens;
+
+import javafx.scene.Group;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.stage.Stage;
+
+/**
+ * Created by Martijn on 24-5-2017.
```
Comment (blazeuser) [inline src/main/java/screens/Factory.java]: Please place proper comments.

File: src/main/java/screens/Factory.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ */
+public class Factory {
+    public Stage createStage() {
```
Comment (blazeuser) [inline src/main/java/screens/Factory.java]: Missing javadoc comment.

File: src/main/java/screens/Factory.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+import javafx.scene.Group;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.stage.Stage;
+
+/**
+ * Created by Martijn on 24-5-2017.
+ */
+public class Factory {
```
Comment (DexterQvL) [inline src/main/java/screens/Factory.java]: Factory is a rather ambiguous name, could you change the name to give more of an indication what this factory produces?

File: src/main/java/screens/Factory.java (hunk 2 above)
Comment (DexterQvL) [inline src/main/java/screens/Factory.java]: All of the methods seem to be missing Javadoc comments, could you add those?

File: src/main/java/screens/Factory.java [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+import javafx.scene.Group;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.stage.Stage;
+
+/**
+ * Factory for JavaFX constructors.
+ */
+public class Factory {
```
Comment (DexterQvL) [inline src/main/java/screens/Factory.java]: Factory is a rather ambiguous name, could you give this class a more descriptive name?

**Pull request merged**